turbo.json

```diff
@@ -8,34 +8,21 @@
   ],
   "pipeline": {
     "site#build": {
-      "dependsOn": [
-        "^build"
-      ],
-      "env": [
-        "BASE_SITE_URL"
-      ],
-      "outputs": [
-        ".next/**"
-      ]
+      "dependsOn": ["^build"],
+      "env": ["BASE_SITE_URL"],
```

Comment:
Added because of this error: https://github.com/vercel/turborepo/blob/main/packages/eslint-plugin-turbo/docs/rules/no-undeclared-env-vars.md